checkout/build.gradle.kts

```
@@ -0,0 +1,96 @@
+import com.mercadopago.sdk.android.CheckoutSDKConfig
+
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.klint)
+    alias(libs.plugins.kotlin.compose)
+    alias(libs.plugins.google.ksp)
+    id(MavenConfig.MAVEN_PUBLISH)
+}
+
+publishing {
+    publications {
+        register<MavenPublication>(MavenConfig.RELEASE) {
+            groupId = MavenConfig.GROUP_ID
+            artifactId = CheckoutSDKConfig.ARTIFACT_ID
+            version = CheckoutSDKConfig.VERSION_NAME
+            afterEvaluate {
+                from(components[MavenConfig.RELEASE])
+            }
+        }
+    }
+    repositories {
+        maven {
+            name = MavenConfig.NEXUS_NAME
+            credentials {
+                username = System.getenv(MavenConfig.USERNAME)
+                password = System.getenv(MavenConfig.PASSWORD)
+            }
+            url = uri(MavenConfig.NEXUS_URL)
+        }
+    }
+}
+
+android {
+    namespace = "com.mercadopago.android.sdk.checkout"
+    compileSdk = MercadoPagoSDKConfig.COMPILE_SDK
+
+    defaultConfig {
+        minSdk = MercadoPagoSDKConfig.MIN_SDK
+        version = CoreMethodsSDKConfig.VERSION_NAME
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
```

could we set it as true by default? this way, we won’t forget about it once we make it public
```suggestion
            isMinifyEnabled = true
```